### PR TITLE
Port #209 to 1.x as suggested in #221

### PR DESCRIPTION
**Pull request by mikey179**
This ports the namespace change introduced with #209 to the 1.x series, as suggested in #221. This allows users to upgrade to 1.7 and adjust the used namespace.

This doesn't contain the name changes for the classes and interfaces yet. I'll introduce those in a separate PR once this gets approved and merged.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,13 +11,6 @@ php:
 
 matrix:
   fast_finish: true
-  include:
-    - php: 5.5
-      dist: precise
-    - php: 5.4
-      dist: precise
-    - php: 5.3
-      dist: precise
```
Comment (allejo) [inline .travis.yml]: If we're dropping 5.3 - 5.5 support in this PR for the 1.7.x series, then we should bump the composer.json as well.

https://github.com/bovigo/vfsStream/blob/926040619cb9d084aee75a2da1cd5cd61f33c39a/composer.json#L20

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+=======
+1.7.0 (20??-??-??)
+------------------
+
+  * Namespace changed from `org\bovigo\vfs` to `bovigo\vfs`
+    - The old namespace still works, but has been deprecated. It will be removed in version 2.
```
Comment (allejo) [inline CHANGELOG.md]: Document the bump to 5.6 as a PHP minimum version

File: .travis.yml (hunk 1 above)
Comment (mikey179) [reply .travis.yml]: You are right, just adjusted that. Anybody still running these PHP versions has larger problems than upgrading to the next version of vfsStream, I guess, so it should be safe to drop support for those.

File: CHANGELOG.md (hunk 2 above)
Comment (mikey179) [reply CHANGELOG.md]: Point added.

**Pull request merged**